packages/scheduled-tasks/src/lib/structures/ScheduledTaskStore.ts

```diff
@@ -0,0 +1,34 @@
+import { Store } from '@sapphire/pieces';
+import { ScheduledTask } from './ScheduledTask';
+
+export class ScheduledTaskStore extends Store<ScheduledTask> {
+	public readonly repeatedTasks: ScheduledTask[] = [];
+
+	public constructor() {
+		super(ScheduledTask as any, { name: ScheduledTaskStore.storeName });
+	}
+
+	public set(key: string, value: ScheduledTask): this {
+		if (value.interval !== null || value.cron !== null) {
+			this.repeatedTasks.push(value);
+		}
+
+		return super.set(key, value);
+	}
+
+	public delete(key: string): boolean {
+		const index = this.repeatedTasks.findIndex((task) => task.name === key);
+
+		// If the middleware was found, remove it
+		if (index !== -1) this.repeatedTasks.splice(index, 1);
+
+		return super.delete(key);
+	}
+
+	public clear(): void {
+		this.repeatedTasks.length = 0;
+		return super.clear();
+	}
+
+	public static storeName = 'scheduled-tasks';
```

Comment:
Do we do this anywhere else in Sapphire? Or do we just inline them into the name parameter?

Reply:
We don't do it anywhere else in Sapphire, for example: https://github.com/sapphiredev/plugins/blob/62a0ad03993f2008234268bec0d855afae8e166b/packages/api/src/lib/structures/MediaParserStore.ts#L7-L10

Reply:
Should I do something with it or should I leave it this way?

Reply:
Change it, to match the MediaParserStore code above. So no static property, just inline it.